scipy/integrate/tests/test_quadrature.py

```diff
@@ -265,32 +266,34 @@ def test_zero_len_y(self):
 
 
 class TestTrapezoid:
-    def test_simple(self):
-        x = np.arange(-10, 10, .1)
-        r = trapezoid(np.exp(-.5 * x ** 2) / np.sqrt(2 * np.pi), dx=0.1)
+    @array_api_compatible
```

Comment:
you can apply `@array_api_compatible` just once to the `TestTrapezoid` class rather than to each method since all tests have been converted. (you'll need to add an unused `xp` arg to `test_masked` I think, but that's fine)